frontend/src/utils/TaskUtils.ts

```diff
@@ -220,51 +220,64 @@ export const dragDropBetweenLists = (
   };
 };
 
-export const calcTaskWeightedValueSum = (
-  task: Task,
-  allCustomers: Customer[],
-  roadmap: Roadmap,
+const ratingValue = (allCustomers: Customer[], roadmap: Roadmap) => (
+  rating: Taskrating,
 ) => {
-  const customerValuesSum = allCustomers.reduce(
-    (total, customer) => total + customerWeight(customer),
-    0,
+  const ratingCreator = allCustomers.find(
+    ({ id }) => id === rating.forCustomer,
   );
+  const creatorWeight = ratingCreator
+    ? customerWeight(ratingCreator, roadmap.plannerCustomerWeights)
+    : 0;
 
-  const ratingValue = (rating: Taskrating) => {
-    const ratingCreator = allCustomers.find(
-      ({ id }) => id === rating.forCustomer,
-    );
-
-    const ratingCreatorValue = ratingCreator
-      ? customerWeight(ratingCreator, roadmap.plannerCustomerWeights)
-      : 0;
-
-    let creatorPlannerWeight = roadmap.plannerCustomerWeights?.find(
-      ({ customerId }) => customerId === rating.forCustomer,
-    )?.weight;
-    if (creatorPlannerWeight === undefined) creatorPlannerWeight = 1;
+  return rating.value * creatorWeight;
+};
 
-    let creatorValueWeight = 0;
-    if (ratingCreator === undefined) creatorValueWeight = 1;
+export const taskWeightedValueSummaryInto = (
```

Comment:
Is there a need for both this and `taskWeightedValueSummary`?

Also both are only used internally, so maybe these could be not exported, unless there is use case in sight

Reply:
I guess it was rather useless to have taskWeightedValueSummary to only provide the RatingsSummary. Removed!
Removed the export from the other function as well